src/develop/blend.c

```diff
@@ -213,9 +213,9 @@ static inline void _Lab_2_LCH(const float *Lab, float *LCH)
   float var_H = atan2f(Lab[2], Lab[1]);
 
   if(var_H > 0.0f)
-    var_H = var_H / (2.0f * M_PI);
+    var_H = var_H / (2.0f * 3.141592654f);
   else
-    var_H = 1.0f - fabs(var_H) / (2.0f * M_PI);
+    var_H = 1.0f + var_H / (2.0f * 3.141592654f);
 
```

Comment:
What's the rationale for this change?

Reply:
The variable `var_H` is negative in the `else` of `if(var_H > 0.0f)`. Therefore, `1.0f - fabs(var_H)/ (2.0f * M_PI)` is equivalent to `1.0f + var_H / (2.0f * M_PI)` in this particular context.